govc/test/vm_snapshot.bats

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bats
+
+load test_helper
+
+test_vm_snapshot() {
```

Comment:
ok to call the file and the func/test names just "snapshot" without the "vm" prefix, since the command doesn't have it either.